components/dashboard/src/settings/Notifications.tsx

```diff
@@ -1,9 +1,62 @@
+import { useContext } from "react";
 import { SettingsPage } from "./SettingsPage";
+import { getGitpodService } from "../service/service";
+import { UserContext } from "../user-context";
 
 export default function Notifications() {
+    const ctx = useContext(UserContext);
+    const user = ctx.user;
+    const isTransactionalMail = !user?.additionalData?.emailNotificationSettings?.disallowTransactionalEmails;
+    const toggleTransactionalMail = async ()=>{
+        if (user) {
+            user.additionalData = {
+                ... {
+                    ... user.additionalData,
+                    emailNotificationSettings: {
+                        ... user.additionalData?.emailNotificationSettings,
+                        disallowTransactionalEmails: !!isTransactionalMail
+                    }
+                }
+            }
+            await getGitpodService().server.updateLoggedInUser({
+                additionalData: user.additionalData
+            });
+            ctx.setUser(user);
+        }
+    };
+    const isMarketingMail = !!user?.allowsMarketingCommunication;
+    const toggleMarketingMail = async ()=> {
+        if (user) {
+            user.allowsMarketingCommunication = !user?.allowsMarketingCommunication;
+            await getGitpodService().server.updateLoggedInUser({
+                allowsMarketingCommunication: user.allowsMarketingCommunication
+            });
+            ctx.setUser(user);
+        }
+    }
     return <div>
-        <SettingsPage title='Notifications' subtitle='Email notification preferences'>
-            <h3>Notifications</h3>
+        <SettingsPage title='Notifications' subtitle='Choose when to be notified'>
+            <h3>Email Notification Preferences</h3>
```

Comment:
```suggestion
            <h3>Email notification preferences</h3>
```

Reply:
I think we are moving forward with Title Case, right?